lib/make-disk-image.nix

```diff
@@ -54,7 +54,6 @@ let
     }}/registration
 
     ${systemToInstall.config.system.build.diskoScript}
-    ${systemToInstall.config.system.build.nixos-install}/bin/nixos-install --system ${systemToInstall.config.system.build.toplevel} --keep-going --no-channel-copy -v --no-root-password --option binary-caches ""
```

Comment:
this removes the system installation from the pure builder?

Reply:
yeah you are right, I missed that. I hope this is fine: https://github.com/nix-community/disko/compare/1f5b94b7bc9609cba02369b5a6cb03363c11968a..2351928a9b38ebb55b7bbf61c8d95fb5c2c8af9b